static/stils.css

```diff
@@ -150,3 +150,75 @@ a:hover, a:active, a:focus {
   border: 1px solid rgb(248, 160, 160);
 }
 
+.dizaini {
+  position: relative;
+  display: inline-block;
+}
+
+.dizainpoga {
+  background-color: rgb(76, 80, 76);
+  color: white;
+  padding: 6px;
+  font-size: 16px;
+  border: none;
+}
+
+.dizaini-saturs {
+  display: none;
+  position: relative;
+  background-color: #f1f5fc;
+  min-width: 225px;
+  z-index: 1;
+}
+.dizaini:hover .dizaini-saturs {
+  display: block;
+}
+
+.tumss {
+  background-color: rgb(32, 30, 30);
+  color: whitesmoke;
+}
+
+.tumss .chat li.left .chat-body {
```

Comment:
Šo te un to, kas atrodas 187 rindā var apvienot:
```css
.tumss .chat li.left .chat-body, .tumss .chat li.left .chat-body:before {
  background-color: rgb(56, 55, 55);
  border: 1px rgb(56, 55, 55);
}
```
Tas pats ar pārējiem dizainiem.

Reply:
Tā arī izdarīšu